src/secp256k1.c

```diff
@@ -132,6 +138,7 @@ secp256k1_context* secp256k1_context_preallocated_create(void* prealloc, unsigne
     if (flags & SECP256K1_FLAGS_BIT_CONTEXT_VERIFY) {
         secp256k1_ecmult_context_build(&ret->ecmult_ctx, &prealloc);
     }
+    ret->declassify = !!(flags & SECP256K1_FLAGS_BIT_CONTEXT_DECLASSIFY);
```

Comment:
@gmaxwell Can you reply to this one? Ignoring it by clicking "resolve conversation" counts as a reply. :)

Reply:
lol Github was helpfully hiding this one from me, I only found it because you said two... :)

I don't like having an apparently boolean variable have its truth value be other than one.  It would totally be fine to not !! here with the current code,  but then later we go and refactor something and the boolean test gets turned into an ==1 or used in an arithmetic expression and it works incorrectly.

So in general, I usually tend to boolenize  a boolean variable unless its never used outside of a single scope, or in some performance critical inner loop where the single test opcode might actually matter.  Though for flags the more ordinary way to booleanize is (flags & flag) == flag.

Reply:
Nevermind, I got the semantics wrong here.